mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -213,6 +213,22 @@ def process_code_coverage(key, value, timestamp):
 
         p = start_conn_process()
 
+        conn_process_started = False
+        try:
+            (key, value, timestamp) = event_queue.get(timeout=self.options.process_start_timeout)
+
+            if key == '__conn_process_start':
```

Comment:
True! We can definitely add that, though I wasn't sure if we wanted to be tolerant of this kind of "failure" or not.